CONTRIBUTING.md

```diff
@@ -11,4 +13,48 @@ Although contributing to the project is allowed and even encouraged it has to fo
 
 
 
-This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
\ No newline at end of file
+This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
+
+## Installing an editing
+
+### Requirements
+
+You will need python 3 and the library `discord.py`.
```

Comment:
python >=3.7
this is due to f-strings. Also `asyncio` has only been added since 3.5

Reply:
Okay I doubted, So I should put 3.5 or 3.7 ?

Reply:
3.7 yes